sympy/core/function.py

```diff
@@ -2349,7 +2349,7 @@ def count_ops(expr, visual=False):
                     ops.append(o*(len(a.args) - 1))
                 else:
                     ops.append(o)
-            if not a.is_Symbol:
+            if not a.is_Symbol and not a.is_MatrixSymbol:
```

Comment:
I assume this is supposed to catch MatrixElement?

I'm not sure about with your patch but in master, a MatrixElement is not a MatrixSymbol.  So its a bit wierd if `.is_MatrixSymbol` is true...   Possible I've misunderstood.